api/Element.json

```diff
@@ -7164,7 +7164,8 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "6",
+              "notes": "In IE 6 and earlier, <code>setAttribute</code> does not set styles and removes events when you try to set them."
```

Comment:
This note seems a little awkward, specifically "In IE 6 and _earlier_".  Since IE 6 was set as the first version that had support, there's no earlier version to be had.  May I ask where this information is sourced from?

Reply:
of course. 
https://quirksmode.org/dom/core/
>IE5-7 doesn't set styles and removes events when you try to set them